src/i18n/es.js

```diff
@@ -67,6 +67,11 @@ const messages_es = {
     free_title: `Big Query`,
     free_ul_item_filter: `Podras filtrar los reportes en base a todas tus campañas o seleccionar solo algunas en particular.`,
     free_ul_item_insights: `Obtén valiosos insights sobre tu cuenta`,
+    plus_button_add: `+`,
+    plus_button_remove: `-`,
+    plus_error_message_email_exists: `El correo ingresado ya se encuentra configurado`,
+    plus_error_message_empty: `El campo esta vacio`,
```

Comment:
```suggestion
    plus_error_message_empty: `El campo esta vacío`,
```

Reply:
this label not it's necessary, because the button add only is active when field is not empty, I'm going to remove this tag.

Reply:
good catch

Reply:
Hecho, resuelvo la conversación...